### ISSUE TITLE
Where ist the .app?

### ISSUE THREAD

**Issue by rin67630** [suolapeikko/FinderUtilities#7]
Could you please make the readme idiot-proof?

_Just download the application from releases (or compile it from the sources), copy it to your /Applications directory and run it once to install the containing Finder Extension._

I am really sorry, but i could not find any FinderUtilities.app in your project.
Could you please explain which file is supposed to be copied to /Applications?

**Comment by rin67630** [comment]
where is the folder /releases?
https://github.com/suolapeikko/FinderUtilities/releases

**Comment by suolapeikko** [comment]
Download the FinderUtilities.zip from the releases URL above, the app file is inside the zip archive.

**Comment by rin67630** [comment]
Yes I know, but the /releases folder exists, but is not listed in the GitHub project.
Do you know why?
